admin_manual/maintenance/backup.rst

```diff
@@ -2,40 +2,49 @@
 Backing up ownCloud
 ===================
 
-To backup an ownCloud installation there are three main things you need to retain:
+When you backup your ownCloud server, there are four things that you need to copy:
 
-#. The config folder
-#. The data folder
-#. The database
+#. Your ``config/`` directory.
```

Comment:
In general I prefer the more neutral tone and avoid using "you". But if we do it in other parts of the docs then it's fine.
It's just my personal taste :smile: